src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,76 @@
+(function(transport, global, console) {
+    "use strict";
+    
+    function transportSend(args, type) {        
+        // To avoid Runtime.evaluate messages
+        if(args[0] !== "Runtime.evaluate") {
```

Comment:
This is being fixed in https://github.com/mozilla/brackets/pull/628, so you can remove this check altogether.